resources/js/processes-catalogue/components/menuCatologue.vue

```diff
@@ -82,24 +82,52 @@
         </b-list-group-item>
       </b-list-group>
     </b-collapse>
+
+    <select-template-modal
+      :type="$t('Process')"
+      :count-categories="categoryCount"
+      ref="addProcessModal"
+      hide-add-btn="true"
+    >
+    </select-template-modal>
   </div>
 </template>
 
 <script>
 import SearchCategories from "./utils/SearchCategories.vue";
+import SelectTemplateModal from "../../components/templates/SelectTemplateModal.vue";
 
 export default {
   components: {
     SearchCategories,
+    SelectTemplateModal,
   },
-  props: ["data", "select", "title", "preicon", "filterCategories", "fromProcessList"],
+  props: [
+    "data",
+    "select",
+    "title",
+    "preicon",
+    "filterCategories",
+    "fromProcessList",
+    "categoryCount",
```

Comment:
Please use the permission in order to see the new option, this is not necesary

Reply:
categoryCount and permissions are not related. the variable categoryCount is necessary to send a prop to CreateProcessModal component.